integreat_cms/cms/templates/organizations/organization_form.html

```diff
@@ -43,20 +59,85 @@ <h3 class="heading font-bold text-black"><i data-feather="settings" class="mr-2"
 			</div>
 		</div>
 	</div>
-	{% if form.instance.id and perms.cms.delete_organization %}
-	<div class="pt-2 pb-4">
-		<button title="{% trans 'Delete organization' %}" class="btn confirmation-button btn-red"
-				data-confirmation-title="{{ delete_dialog_title }}"
-				data-confirmation-text="{{ delete_dialog_text }}"
-				data-confirmation-subject="{{ form.name.value }}"
-				data-action="{% url 'delete_organization' region_slug=request.region.slug slug=form.instance.slug %}">
-			<i data-feather="trash-2" class="mr-2"></i>
-			{% trans 'Delete this organization' %}
-		</button>
+	{% if form.instance.id %}
+
+	{% get_current_language as LANGUAGE_CODE %}
+	{% get_language LANGUAGE_CODE as backend_language %}
+	<div class="grid xl:grid-cols-2 gap-4">
+		<div class="mb-4 rounded border border-solid border-blue-500 shadow-2xl bg-white">
+			<div class="p-4 rounded bg-water-500">
+			<h3 class="heading font-bold text-black"> <i data-feather="layout" class="mr-2"></i>
+				{% trans "Pages" %}
+			</h3>
+			</div>
+			<div class="table-listing w-full p-2">
+				<table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+					<thead>
+						<tr class="border-b border-solid border-gray-200">
+							<th class="text-sm text-left uppercase py-3 pl-4 pr-2">
+								{% trans "Name in Englisch" %}
```

Comment:
> How to display the pages in the organization edit page (what information, should we add a language selecting gadget here as well?)

I think a language selector would be overkill, but maybe two columns: The region's default language and the current backend UI language? (And if both are identical, then just one column?)

Reply:
do we want to add a redirect to the pages somewhere or just simply display them?

Reply:
Yes, good idea :+1:
Maybe just make the table cells link to the page form (similar to the regular page tree)?

Reply:
Yes, thats what I had in mind as well  :+1: